lib/std/Target/csky.zig

```diff
@@ -214,7 +214,7 @@ pub const all_features = blk: {
     };
     result[@intFromEnum(Feature.dsp_silan)] = .{
         .llvm_name = "dsp_silan",
-        .description = "Enable DSP Silan instrutions",
+        .description = "Enable DSP Silan instructions",
```

Comment:
Not a merge blocker but this data comes from LLVM so you may want to submit this change upstream as well.